org.adempiere.base/src/org/compiere/process/SvrProcess.java

```diff
@@ -762,19 +762,21 @@ private void autoFillParameters(){
             String name = parameter.getParameterName().trim().toLowerCase();
             Field field = map.get(name);
             Field toField = map.containsKey(name + "_to") ? map.get(name + "_to") : null;
+            Field notField = map.containsKey(name + "_not") ? map.get(name + "_not") : null;
 
             // try to match fields using the "p_" prefix convention
             if(field==null) {
             	String candidate = "p_" + name;
                 field = map.get(candidate);
                 toField = map.containsKey(candidate + "_to") ? map.get(candidate + "_to") : null;
+                notField = map.containsKey(candidate + "_not") ? map.get(candidate + "_not") : null;
             }
 
             // try to match fields with same name as metadata declaration after stripping "_"
             if(field==null) {
             	String candidate = name.replace("_", "");
                 field = map.get(candidate);
-                toField = map.containsKey(candidate + "to") ? map.get(candidate + "to") : null;
+                notField = map.containsKey(candidate + "to") ? map.get(candidate + "to") : null;

```

Comment:
this is wrong, should be the same as line 765 above, i.e keep toField and add notField for candidate + "not"

Reply:
Yes, its Wrong @hengsin I will make the correction